assets/src/data/model/entity-factory/create.js

```diff
@@ -0,0 +1,701 @@
+/**
```

Comment:
This file basically contains all the functions used to create various things on BaseEntity.